src/components/dashboard/StackingStats.js

```diff
@@ -24,7 +24,7 @@ export default function StackingStats(props) {
         } else {
           if (endBlock - blockHeight < props.config.rewardCycleLength) {
             setPercentComplete(
-              Math.round(((endBlock - blockHeight) / props.config.rewardCycleLength) * 100)
+              Math.round(100 - ((endBlock - blockHeight) / props.config.rewardCycleLength) * 100)
```

Comment:
```suggestion
              Math.round((1 - (endBlock - blockHeight) / props.config.rewardCycleLength) * 100)
```
This subtraction from 100 feels a bit weird. I always subtract from 1 and then multiply by 100.
Nevertheless result should be the same.